ruler-frontend/src/main/kotlin/com/spotify/ruler/frontend/components/Insights.kt

```diff
@@ -40,6 +41,9 @@ fun RBuilder.insights(components: List<AppComponent>) {
     div(classes = "row") {
         componentTypeGraphs(components)
     }
+    div(classes = "row") {
+        resourcesTypeGraph(components)
```

Comment:
Nit - for consistency this should be called `resourceTypeGraphs` (resource in singular and graphs in plural) 😬